src/vaadin-number-field.html

```diff
@@ -289,8 +302,9 @@
 
         checkValidity() {
           // text-field mixin does not check against `min` and `max`
-          if (this.min !== undefined || this.max !== undefined) {
-            this.invalid = !this.inputElement.checkValidity();
+          if (this.value && (this.min !== undefined || this.max !== undefined || this.step)) {
+            const stepBase = this.min || 0;
+            this.invalid = this.value < this.min || this.value > this.max || (this.value - stepBase).toFixed(this.__decimals) % this.step !== 0;
```

Comment:
According to https://caniuse.com/#feat=input-number iOS 9 Safari wouldn't take min/max/step into account in validation, but just tested it manually and it _seems_ to work. See if we could after all just rely on the native input's validation.

Reply:
Ok I see. I changed back to using the native validation. Will the Travis tests catch if this doesn't work?